### ISSUE TITLE
E-Mail report is empty when using Gmail app

### ISSUE THREAD

**Issue by m-rm** [ACRA/acra#744]
I have implemented ACRA in my app a week ago and it worked fine. Now when the e-mail app (Gmail) is opened all fields are empty and no crash report is attached. I reverted back to a previous working commit and still have the same issue. So, I guess it's not directly related to the library but rather the Gmail app or the Android OS itself.

Configuration:
```
@AcraCore(buildConfigClass = BuildConfig::class)
@AcraMailSender(mailTo = Constants.CRASHREPORT_EMAIL,
        resSubject = R.string.crash_subject,
        reportAsFile = true)
@AcraDialog(resText = R.string.crash_text,
        resTitle = R.string.crash_title,
        resPositiveButtonText = R.string.crash_button_positive,
        resNegativeButtonText = R.string.crash_button_negative,
        resTheme = R.style.AppThemeNoActionBar)
```

There are only 2 lines in logcat that might be interesting:

> 2019-05-03 10:29:58.564 12643-12643/my.package.name:acra W/my.package.name:acra: type=1400 audit(0.0:186): avc: denied { read } for comm=45474C20496E6974 name="u:object_r:vendor_default_prop:s0" dev="tmpfs" ino=16933 scontext=u:r:untrusted_app:s0:c133,c257,c512,c768 tcontext=u:object_r:vendor_default_prop:s0 tclass=file permissive=0
> 2019-05-03 10:30:00.017 12643-12714/my.package.name:acra I/ACRA: Sending report /data/user/0/my.package.name/app_ACRA-approved/2019-05-03T10:29:57.735+02:00.stacktrace

I'm using a Pixel 2 with Android 9.0 with ACRA version 5.3.0.

**Comment by F43nd1r** [comment]
Turn on ACRA.DEV_LOGGING and post a full log

**Comment by m-rm** [comment]
Here you go:
```2019-05-06 11:55:01.048 3059-3059/? W/nect.debug:acr: Accessing hidden method Landroid/app/AppGlobals;->getInitialApplication()Landroid/app/Application; (light greylist, reflection)
2019-05-06 11:55:01.118 3059-3100/? D/libEGL: loaded /vendor/lib64/egl/libEGL_adreno.so
2019-05-06 11:55:01.122 3059-3100/? E/libc: Access denied finding property "vendor.debug.egl.profiler"
2019-05-06 11:55:01.118 3059-3059/? W/my.package.name:acra: type=1400 audit(0.0:3086): avc: denied { read } for comm=45474C20496E6974 name="u:object_r:vendor_default_prop:s0" dev="tmpfs" ino=16933 scontext=u:r:untrusted_app:s0:c134,c257,c512,c768 tcontext=u:object_r:vendor_default_prop:s0 tclass=file permissive=0
2019-05-06 11:55:01.124 3059-3100/? D/libEGL: loaded /vendor/lib64/egl/libGLESv1_CM_adreno.so
2019-05-06 11:55:01.136 3059-3100/? D/libEGL: loaded /vendor/lib64/egl/libGLESv2_adreno.so
2019-05-06 11:55:01.141 3059-3059/? D/ACRA: CrashReportDialog extras=Bundle[mParcelledData.dataSize=3600]
2019-05-06 11:55:01.176 3059-3059/? D/ACRA: Checking plugin Configurations : ImmutableList{[org.acra.config.DialogConfiguration@803f26, org.acra.config.MailSenderConfiguration@73ca767]} for class : class org.acra.config.DialogConfiguration
2019-05-06 11:55:01.176 3059-3059/? D/ACRA: Checking plugin Configuration : org.acra.config.DialogConfiguration@803f26 against plugin class : class org.acra.config.DialogConfiguration
2019-05-06 11:55:01.214 3059-3059/? D/OpenGLRenderer: Skia GL Pipeline
2019-05-06 11:55:01.265 3059-3105/? I/Adreno: QUALCOMM build                   : 6fb5a5b, Ife855c4895
    Build Date                       : 08/21/18
    OpenGL ES Shader Compiler Version: EV031.25.00.00
    Local Branch                     : Googledrop_0815
    Remote Branch                    : 
    Remote Branch                    : 
    Reconstruct Branch               : 
2019-05-06 11:55:01.266 3059-3105/? I/Adreno: Build Config                     : S L 4.0.10 AArch64
2019-05-06 11:55:01.268 3059-3105/? I/Adreno: PFP: 0x005ff110, ME: 0x005ff066
2019-05-06 11:55:01.271 3059-3105/? I/ConfigStore: android::hardware::configstore::V1_0::ISurfaceFlingerConfigs::hasWideColorDisplay retrieved: 1
2019-05-06 11:55:01.271 3059-3105/? I/ConfigStore: android::hardware::configstore::V1_0::ISurfaceFlingerConfigs::hasHDRDisplay retrieved: 0
2019-05-06 11:55:01.271 3059-3105/? I/OpenGLRenderer: Initialized EGL, version 1.4
2019-05-06 11:55:01.271 3059-3105/? D/OpenGLRenderer: Swap behavior 1
2019-05-06 11:55:34.304 3059-3121/my.package.name:acra D/ACRA: Add user comment to /data/user/0/my.package.name/app_ACRA-unapproved/2019-05-06T11:55:00.337+02:00.stacktrace
2019-05-06 11:55:34.325 3059-3105/my.package.name:acra D/OpenGLRenderer: endAllActiveAnimators on 0x70295e7500 (RippleDrawable) with handle 0x7035e9fe40
2019-05-06 11:55:34.331 3059-3121/my.package.name:acra D/ACRA: ServicePluginLoader loading services from ServiceLoader : java.util.ServiceLoader[org.acra.scheduler.SenderSchedulerFactory]
2019-05-06 11:55:34.334 3059-3121/my.package.name:acra D/ACRA: Found services ([]) for class : interface org.acra.scheduler.SenderSchedulerFactory
2019-05-06 11:55:34.337 3059-3121/my.package.name:acra D/ACRA: Mark 2019-05-06T11:55:00.337+02:00.stacktrace as approved.
2019-05-06 11:55:34.338 3059-3121/my.package.name:acra D/ACRA: Schedule report sending
2019-05-06 11:55:34.392 3059-3128/my.package.name:acra D/ACRA: About to start sending reports from SenderService
2019-05-06 11:55:34.392 3059-3128/my.package.name:acra D/ACRA: config#reportSenderFactoryClasses : ImmutableList{[]}
2019-05-06 11:55:34.392 3059-3128/my.package.name:acra D/ACRA: Using PluginLoader to find ReportSender factories
2019-05-06 11:55:34.392 3059-3128/my.package.name:acra D/ACRA: ServicePluginLoader loading services from ServiceLoader : java.util.ServiceLoader[org.acra.sender.ReportSenderFactory]
2019-05-06 11:55:34.405 3059-3128/my.package.name:acra D/ACRA: Checking plugin Configurations : ImmutableList{[org.acra.config.DialogConfiguration@18823df, org.acra.config.MailSenderConfiguration@f5f682c]} for class : class org.acra.config.MailSenderConfiguration
2019-05-06 11:55:34.405 3059-3128/my.package.name:acra D/ACRA: Checking plugin Configuration : org.acra.config.DialogConfiguration@18823df against plugin class : class org.acra.config.MailSenderConfiguration
2019-05-06 11:55:34.405 3059-3128/my.package.name:acra D/ACRA: Checking plugin Configuration : org.acra.config.MailSenderConfiguration@f5f682c against plugin class : class org.acra.config.MailSenderConfiguration
2019-05-06 11:55:34.405 3059-3128/my.package.name:acra D/ACRA: Loaded ReportSenderFactory of type org.acra.sender.EmailIntentSenderFactory
2019-05-06 11:55:34.406 3059-3128/my.package.name:acra D/ACRA: Found services ([org.acra.sender.EmailIntentSenderFactory@b64f2f5]) for class : interface org.acra.sender.ReportSenderFactory
2019-05-06 11:55:34.406 3059-3128/my.package.name:acra D/ACRA: reportSenderFactories : [org.acra.sender.EmailIntentSenderFactory@b64f2f5]
2019-05-06 11:55:34.407 3059-3128/my.package.name:acra D/ACRA: Checking plugin Configurations : ImmutableList{[org.acra.config.DialogConfiguration@18823df, org.acra.config.MailSenderConfiguration@f5f682c]} for class : class org.acra.config.MailSenderConfiguration
2019-05-06 11:55:34.407 3059-3128/my.package.name:acra D/ACRA: Checking plugin Configuration : org.acra.config.DialogConfiguration@18823df against plugin class : class org.acra.config.MailSenderConfiguration
2019-05-06 11:55:34.407 3059-3128/my.package.name:acra D/ACRA: Checking plugin Configuration : org.acra.config.MailSenderConfiguration@f5f682c against plugin class : class org.acra.config.MailSenderConfiguration
2019-05-06 11:55:34.408 3059-3128/my.package.name:acra D/ACRA: Adding reportSender : org.acra.sender.EmailIntentSender@a80488a
2019-05-06 11:55:34.409 3059-3128/my.package.name:acra I/ACRA: Sending report /data/user/0/my.package.name/app_ACRA-approved/2019-05-06T11:55:00.337+02:00.stacktrace
2019-05-06 11:55:34.437 3059-3128/my.package.name:acra D/ACRA: Sending report using org.acra.sender.EmailIntentSender
2019-05-06 11:55:34.465 3059-3128/my.package.name:acra D/ACRA: Sent report using org.acra.sender.EmailIntentSender
2019-05-06 11:55:34.465 3059-3128/my.package.name:acra D/ACRA: Report was sent by all senders
2019-05-06 11:55:34.465 3059-3128/my.package.name:acra D/ACRA: Finished sending reports from SenderService```

**Comment by m-rm** [comment]
I installed K9 Mail additionally to see if it's a problem with Gmail. It works fine with K9 Mail, so the Gmail app ist definitly the problem. I checked logcat again without any application filter and found this everytime the crash report should be sent:
```
2019-05-06 16:27:19.434 19330-19330/? E/Gmail: ComposeActivity:Something failed while loading ConversationMessageDraft.
    java.lang.NullPointerException: Attempt to invoke interface method 'java.lang.String java.lang.CharSequence.toString()' on a null object reference
        at dgr.a(SourceFile:634)
        at dgw.a(Unknown Source:7)
        at abhi.a(Unknown Source:5)
        at adiv.a(SourceFile:2)
        at adit.run(SourceFile:6)
        at adki.run(Unknown Source:7)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:458)
        at abfe.run(SourceFile:2)
        at zve.run(Unknown Source:3)
        at android.os.Handler.handleCallback(Handler.java:873)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6718)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
```

**Comment by F43nd1r** [comment]
Very weird. Is this limited to one device?

**Comment by m-rm** [comment]
I tried 2 more devices today: Another Pixel 2 with Android 9 and a Samsung SM-T585 Tablet with 8.1. Same issue. So I deactivated Gmail (which resets it back to the stock version) and installed an older version. This made it work again. So I tested the last 6 available versions on APKMirror and came up with this:

2019.04.14.246198419 | Working again, but is not (yet) on Play Store
2019.03.31.243845549 | Broken, this is the current version on Play Store
2019.03.31.243253844 | Broken
2019.03.31.243129570 | Broken
2019.03.17.241582812 | Working
2019.03.17.240613483 | Working
